src/styles/product-card.css

```diff
@@ -0,0 +1,101 @@
+.card {
+  width: 200px;
+  height: 406px;
```

Comment:
```suggestion
```

It would be nice not to hardcode card height in this task (optional)

Reply:
So, i need to change harcored width to padding-bottom?

Reply:
Yes, use paddings and margins